package.json

```diff
@@ -168,7 +168,6 @@
     "sentry": "npx -p @sentry/cli sentry-cli",
     "update-browserslist": "npx browserslist@latest -- --update-db",
     "prepare": "husky install",
-    "format": "prettier --ignore-unknown .",
     "format-fix": "prettier --write .",
```

Comment:
Can we have `format` be the one that fixes it and drop the `-fix` naming?

Reply:
Yes! I'd say go for it. The original names are just me making names up as I went, so I think fixing those would be great

Reply:
Agreed here - change both please :)